### DISCUSSION TITLE
The file mode setting in Day Planner

### DISCUSSION THREAD

**Discussion by Zhuguli1225** [ivan-lednev/obsidian-day-planner#453] [Q&A]
My Day Planner plugin is 0.20.4, and I cannot find the file mode setting in the settings. I want to change the file mode to command mode, will someone help me ?

**Comment by ivan-lednev** [comment]
Hello, @Zhuguli1225, do you mean you want to be able to pin the planner to a specific file with a command?

**Reply by cacosat** [reply]
How is this done?

**Reply by ivan-lednev** [reply]
This feature is currently not available.
